### DISCUSSION TITLE
ssl issue with heroku [Apr 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1667] [Discord Threads]
> 🙋‍♂️ **sangle**  
#### Sang.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/960314332518965258/Screen_Shot_2022-04-04_at_06.02.10.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Edit Connection JSON and configure ssl   
####    
#### ```   
####      "ssl": {   
####       "require": true,   
####       "rejectUnauthorized": false,   
####       "sslMode": "no-verify"   
####     }   
#### ```  
#  
> 💁 **o1lab**  
#### https://youtu.be/rS0NRwRCjx0  
#  
> 💁 **o1lab**  
#### Pls see this video on how to edit json. And try <@495466997304262658> suggestions  
#  
> 🙋‍♂️ **sangle**  
#### I am not sure what to config. Could u plz elaborate with me more on this? Tks  
#  
> 💁 **o1lab**  
#### you would have to edit the json like wing has proposed.. and for that json editor can be opened like in the video  
#  
> 🙋‍♂️ **sangle**  
#### Thank you <@495466997304262658> for your help. I made it work 🙂  
#  
> 🙋‍♂️ **sangle**  
#### Thank you o1lab@discord too!  
#  
> 🙋‍♂️ **sangle**  
#### Highly appreciate  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/960314333643030618)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
